string rc_path
string pc_path
---
bool success
string error_msg